### DISCUSSION TITLE
PROBLEMA NO RECEBIMENTO DO EMAIL

### DISCUSSION THREAD

**Discussion by mattheuz-lima** [emailjs-com/help#37] [Q&A] [answered]
Olá! Estou enfrentando um problema ao tentar enviar emails usando o Email.js. Eu segui todas as etapas de configuração corretamente, mas quando clico no botão de enviar, quem recebe o Template Email que foi configurado é o meu próprio email que está cadastrado no Email.js.

Queria saber como faço para que esse Email seja enviado para a pessoa que preencheu o input com seu Email?

**Comment by oculosdanilo** [comment, marked as answer]
pra isso voce vai ter que colocar no site do emailjs que isso é uma variavel (tipo `{{destinatario}}`), salvar e depois informar no código de enviar `"destinatario": "algumEmail@gmail.com"`

declara a variavel aqui:
![image](https://github.com/emailjs-com/help/assets/87551338/eb0260c1-7835-4fc2-89e8-35636ec071e9)
